libraryAPI/pkg/repository/repository.go

```diff
@@ -1,11 +1,13 @@
 package repository
 
+import "library/config"
+
 type Repository struct {
 	BookRepository
 }
 
-func NewRepository() *Repository {
+func NewRepository(c config.Config) *Repository {
```

Comment:
you don't need to provide the whole config here, just the path to the CSV file, also think about a separate method like 'NewConn' and move logic related to opening the file there
Mention: in this case you will defer close conn to file in main.go